includes_custom_resources/includes_custom_resources_syntax.rst

```diff
@@ -5,25 +5,20 @@
 A custom resource is defined as a |ruby| file and is located in a cookbook's ``/resources`` directory. This file
 
 * Declares the properties of the custom resource
-* Loads current properties, if the resource already exists
 * Defines each action the custom resource may take
 
-The syntax for a custom resource is. For example:
+The syntax for a basic custom resource utilizing built-in chef resources is. For example:
```

Comment:
I think we should make it clear that you can use ruby here, and it's safe to do so directly and that chef will take care of running it at the correct time.